lib/commons/text/form-control-value.js

```diff
@@ -106,7 +106,7 @@ function nativeSelectValue(node) {
   }
 
   const options = querySelectorAll(vNode, 'option');
-  const selectedOptions = options.filter(option => option.hasAttr('selected'));
+  const selectedOptions = options.filter(option => option.actualNode.selected);
```

Comment:
I believe this means dependent checks can no longer operate on virtual nodes, which is something we've been pushing for lately. We may need to find a different way to solve this issue, as I don't think we want to cause a regression while fixing a bug.

Reply:
What we'll probably need to do is add `selected` as a [prop](https://github.com/dequelabs/axe-core/blob/develop/lib/core/base/virtual-node/virtual-node.js#L57-L80) of the virtual node class so we can access it that way

```js
const selectedOptions = options.filter(option => option.props.selected);
```

Reply:
Oh I didn't know that 😅. I pushed a change in [`3a71bff` (#3154)](https://github.com/dequelabs/axe-core/pull/3154/commits/3a71bffa0b084356d16bc49a8f7acefebb64b7fa) to use `props.selected` instead. Not sure if my changes make sense though.